packages/odyssey-storybook/src/components/odyssey-mui/Link/Link.stories.tsx

```diff
@@ -62,6 +64,12 @@ Default.args = {
   variant: "default",
   children: "Anchor link",
 };
+Default.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const link = await canvas.getByText("Anchor link");
+  await expect(link).toBeInTheDocument();
+  await userEvent.hover(link);
+};
```

Comment:
Are we also verifying the styles have changed when hovering or that a hover state exists? Or is it taking screenshots somehow?